utils/ort/src/main/kotlin/storage/S3FileStorage.kt

```diff
@@ -86,6 +89,11 @@ class S3FileStorage(
 
             if (customEndpoint != null) {
                 endpointOverride(URI(customEndpoint))
+                serviceConfiguration {
+                    it.pathStyleAccessEnabled(pathStyleAccess)
+                    it.chunkedEncodingEnabled(false)
+                    it.multiRegionEnabled(false)
```

Comment:
I believe these two lines should simply not be added in this commit, as `chunkedEncodingEnabled` is being made configurable in the next commit, and `multiRegionEnabled` is being removed again there altogether.